Resources/Other-Sources/it.lproj/Localizable.strings

```diff
@@ -0,0 +1,44 @@
+"DifferentTemperatureFormat" = "È %@ %@ rispetto %@%@.";
+"PrecipitationChanceFormat" = "%@ possibilità di %@ %@.";
+"SameTemperatureFormat" = "La temperatura %@ %@ rispetto a %@%@.";
+
+"Hotter" = "molto più caldo";
+"Warmer" = "più caldo";
+"Cooler" = "più freddo";
+"Colder" = "molto più freddo";
+"Same" = "è stabile";
+
+"Today" = "oggi";
+"Yesterday" = "ieri";
+"Tonight" = "questa notte";
+"LastNight" = "la scorsa notte";
+"ThisMorning" = "questa mattina";
+"YesterdayMorning" = "ieri mattina";
+"ThisAfternoon" = "questo pomeriggio";
+"YesterdayAfternoon" = "ieri pomeriggio";
+
+"NoneRain" = "";
+"SlightRain" = ", con bassa probabilità di pioggia";
+"GoodRain" = ", con alta probabilità di pioggia";
+"NoneSnow" = ", nessuna possibilità di nevicate";
+"SlightSnow" = ", con bassa probabilità di nevicate";
+"GoodSnow" = ", con alta probabilità di nevicate";
+
+"Updated %@ at %@" = "Aggiornato il %@ alle %@";
+"Updated at %@" = "Aggiornato il %@";
+
+"Checking Weather..." = "Controllo il Meteo...";
+"Update Failed" = "Update non risucito";
```

Comment:
Yep, sorry about that, typo. The sad thing is that I double checked the text twice :sweat: